app/src/main/res/layouts/loginActivity/layout/footer_layout.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:id="@+id/footer"
+    app:layout_constraintBottom_toBottomOf="parent">
+
+    <TextView
+        android:layout_width="match_parent"
+        android:layout_height="35dp"
+        android:background="@color/black"
+        android:gravity="center"
+        android:text="@string/footer_info"
+        android:textColor="@color/white"
+        android:textSize="12sp"
+        tools:ignore="MissingConstraints" />
```

Comment:
Instead of using "tools:ignore", click the button in the picture below to infer some constraints and do some modification for generated constraints. So namespace at line 4 should also be removed.
![image](https://user-images.githubusercontent.com/35082985/187560892-e628f165-f1dc-449a-8aee-462e3c9d81ee.png)